lib/node_modules/@stdlib/stats/base/dists/studentized-range/README.md

```diff
@@ -45,6 +45,8 @@ The namespace contains the following distribution functions:
 
 <div class="namespace-toc">
 
+-   <span class="signature">[`cdf( x, r, v[, nranges=1] )`][@stdlib/stats/base/dists/studentized-range/cdf]</span><span class="delimiter">: </span><span class="description">Studentized range distribution cumulative distribution function (CDF).</span>
```

Comment:
@Planeshifter I think we need to fix the signatures for the Studentized range packages, as we don't document default parameters this way.